addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -123,6 +127,10 @@ def __gather_emissive_factor(blender_material, export_settings):
 
 
 def __gather_emissive_texture(blender_material, export_settings):
+    # An Emission node, alone, should be considered an Unlit material.
+    if not __has_node_of_type(blender_material, bpy.types.ShaderNodeBsdfPrincipled):
```

Comment:
Likewise here, I think the legacy group node won't be able to produce any emission.